src/Axiom/Set.agda

```diff
@@ -140,6 +147,9 @@ record Theory {ℓ} : Type (sucˡ ℓ) where
   -- don't know that there's a set containing all members of a type, which this is equivalent to
   -- _⁻¹_ : (A → B) → Set B → Set A
   -- f ⁻¹ X = {!!}
+  ≡∈ : {X : Set A}{a a' : A} → a ∈ X → a ≡ a' → a' ∈ X
+  ≡∈ {a = a}{.a}a∈X refl = a∈X
+
```

Comment:
I would have thought this lemma is already provided somewhere else, but I didn't see it.  If someone has seen it before, please let me know where.